dev/com.ibm.ws.microprofile.metrics.monitor/src/com/ibm/ws/microprofile/metrics/monitor/MappingTable.java

```diff
@@ -58,7 +58,11 @@ public MappingTable() {
 			{ "connectionpool.%s.destroy.total", "Destroy Count", "connectionpool.destroy.total.description", COUNTER, MetricUnits.NONE, "DestroyCount", null },
 			{ "connectionpool.%s.managedConnections", "Managed Connections Count", "connectionpool.managedConnections.description", GAUGE, MetricUnits.NONE, "ManagedConnectionCount", null },
 			{ "connectionpool.%s.connectionHandles", "Connection Handles Count", "connectionpool.connectionHandles.description", GAUGE, MetricUnits.NONE, "ConnectionHandleCount", null },
-			{ "connectionpool.%s.freeConnections", "Free Connections Count", "connectionpool.freeConnections.description", GAUGE, MetricUnits.NONE,  "FreeConnectionCount", null }
+			{ "connectionpool.%s.freeConnections", "Free Connections Count", "connectionpool.freeConnections.description", GAUGE, MetricUnits.NONE,  "FreeConnectionCount", null },
+			{ "connectionpool.%s.waitTime.total", "Total Wait Time", "connectionpool.waitTime.total.description", GAUGE, MetricUnits.NANOSECONDS,  "WaitTimeDetails", "total" },
+			{ "connectionpool.%s.inUseTime.total", "Total In Use Time", "connectionpool.inUseTime.total.description", GAUGE, MetricUnits.NANOSECONDS,  "InUseTimeDetails", "total" },
+			{ "connectionpool.%s.grantedConnections.total", "Granted Connections", "connectionpool.grantedConnections.total.description", COUNTER, MetricUnits.NONE,  "WaitTimeDetails", "count" },
```

Comment:
Here we are mapping WaitTimeDetails.count to new metrics connectionpool.%s.grantedConnections.total. Since we are updating waitTimeDetails more often than before, WaitTimeDetails.count is effectively the total number of granted connections for all requests in my understanding.